src/test/java/LoginTests.java

```diff
@@ -18,4 +20,30 @@ public static void LoginEmptyEmailPasswordTest () {
         Assert.assertEquals(driver.getCurrentUrl(), url);
         driver.quit();
     }
-}
+
+    @Test
+    public static void  RegisterToKoel() {
+        WebDriver driver = new ChromeDriver();
+        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
+
+        String url = "https://bbb.testpro.io/registration.php";
+        driver.get(url);
+
+        WebElement emailField = driver.findElement(By.cssSelector("[type='email']"));
+        emailField.click();
+        emailField.sendKeys("demo99@class.com");
+
+        WebElement loginField = driver.findElement(By.cssSelector("[type='submit'"));
+        loginField.click();
+
+        // WebElement message = driver.findElement(By.className("h3"));
+        // Assert.assertTrue(message.isDisplayed());
+
+
+        driver.quit();
+
+    }
+
+
+
+    }
```

Comment:
How can I beautify code in IntelliJ:

The formatting shortcuts in Intellij IDEA are :
For Windows : Ctrl + Alt + L.
For Ubuntu : Ctrl + Alt + Windows + L.
For Mac : ⌥ (Option) + ⌘ (Command) + L.

![image](https://user-images.githubusercontent.com/104761482/209774134-9e63bfed-d768-4455-a860-028305c85e17.png)